headers.js

```diff
@@ -7,8 +7,7 @@ const cspMeta = Object.entries({
     "'self'",
     'data:',
     // Explicitly whitelist our KeepKey versions file
-    // TODO: File manually added to IPFS - we need to instead add it to version control and use a persistent URL.
-    'https://bafybeied24gc2ipvlxdbs4v676dwho2l5aafmngrleic3do2czdvgb546u.ipfs.dweb.link/keepKey.json',
+    process.env.REACT_APP_KEEPKEY_VERSIONS_URL,
```

Comment:
suggest moving this to the bottom of the list with the other env vars; otherwise, gtg

Reply:
For sure. Moved in https://github.com/shapeshift/web/pull/1499/commits/201e80141be2695407e01327c1d306df4b0fa7b8.